app/views/clients/edit.html.slim

```diff
@@ -4,6 +4,5 @@
 
 == render 'form'
 
-.row
-  .col-xs-12
-    = link_to t('back'), clients_path, class: 'btn btn-default'
+= form_navigation_btn :show
```

Comment:
you replaced a `back` button with `show` & `back`.
I think the show button is here not needed, right?
and I didn't understand where this `show` button in the application helper is defined as the others are